.github/bot-action/action.yml

```diff
@@ -0,0 +1,15 @@
+name: 'Strawberry Bot Action'
+inputs:
+  pr_number:
+    required: true
+  status:
+    required: true
+  change_type:
+    required: true
+  changelog:
+    required: true
+  release_card_url:
+    required: false
+runs:
+  using: 'docker'
+  image: 'Dockerfile'
```

Comment:
My plan is to move this action to the strawberry-bot repo and allow multiple "commands", for now it only supports adding a release comment so that's why almost everything is optional.

When moving it to the new repo will make sure to add validation and to support multiple commands :)